cranelift/codegen/src/machinst/abi_impl.rs

```diff
@@ -676,6 +699,7 @@ impl<M: ABIMachineImpl> ABIBody for ABIBodyImpl<M> {
             }
             &ABIArg::Stack(off, mut ty, ext) => {
                 let from_bits = ty_bits(ty) as u8;
+                let off = i32::try_from(off).expect("Argument stack offset greater than 2GB");
```

Comment:
Should we add "not yet implemented" to this error message? This may happen in real-world programs. Alternatively, should we report an `Err` `Result` here, with an impl limit exceeded error?

Reply:
The stack offset here is limited by the stack-frame size, which we clamp to 128 MB in `compute_arg_locs()`, so I don't think we need to bubble up a `Result` (since it should not be reachable code for any input); but I've clarified the error message. Thanks!